GraphClientFactory.md

```diff
@@ -18,7 +18,14 @@ There are two primary usage scenarios for the client factory.  Developers who wi
   - Set the SdkVersion header with the appropriate moniker based on the following structure `graph-{lang}-{version}`.  
 - Enable the developer to select a supported sovereign cloud using an enumerated list.  Selecting the sovereign cloud should ensure that the AuthenticationProvider uses the appropriate Authentication Endpoint.
 - Enable a developer to configure a HTTP proxy that will be used for outgoing requests.
-
+- Enable a developer provide custom URLs or endpoints. 
```

Comment:
we really only care about the hostname here, could we update properties names + language to reflect that? I don't think we need to care about the path or the scheme.

Reply:
Agreed.  We should focus only on host name.